blog/_posts/2021-07-06-distributed-rdataframe.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "RDataFrame is going distributed!"
+layout: archive
+author: Vincenzo Eduardo Padulano
+---
+
+A recently introduced Python package enables distributing ROOT RDataFrame workloads to a set of remote resources.
+This feature is available in experimental phase since the latest ROOT 6.24 release, allowing users to write and run
+their applications from within the same interface while steering the computations to an Apache Spark cluster.
+
+## In detail
+
+[RDataFrame](https://root.cern/doc/master/classROOT_1_1RDataFrame.html) is ROOT's high-level interface for data
+analysis and since its release in 6.14 has seen tons of usages in real world analysis use cases and other examples in
+the wild. Parallelism has always been a staple of its design with support for executing the event loop on all cores
+of a machine thanks to implicit multi-threading. Since ROOT 6.24, this aspect of RDataFrame has been enhanced further
+with distributed computing capabilities, allowing users to run their analysis on multi-node computing clusters through
+widely used frameworks such as [Apache Spark](https://spark.apache.org/), [Dask](https://dask.org/) or
+[AWS Lambda](https://aws.amazon.com/lambda/). Here is an example usage of the tool (requires the Python `pyspark`
+package):
+
+```python
+import ROOT
+ 
+# Point RDataFrame calls to the Spark specific RDataFrame
+RDataFrame = ROOT.RDF.Experimental.Distributed.Spark.RDataFrame
+ 
+# It still accepts the same constructor arguments as traditional RDataFrame
+df = RDataFrame("mytree", "myfile.root")
```

Comment:
Shouldnt' we at least say that this will run in a local Spark process by default, but it's possible to configure the RDataFrame instance to use a preexisting Spark cluster?